### PR TITLE
create task_5

### PR DESCRIPTION
**Pull request by Killerkip10**

### CODE REVIEW

File: task_5/config/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  port: 3000,
```
Comment (NikitaSedyx) [inline task_5/config/index.js]: config should be hardcoded, plus there are packages to manage settings on different levels - file, env vars etc

File: task_5/src/app/news/get.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module.exports = ({ newsMapper }) => {
```
Comment (NikitaSedyx) [inline task_5/src/app/news/get.js]: why not 2 files?

File: task_5/src/app/news/post.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+const { trim } = require('lodash');
+const { News: toEntity } = require('../../domain/news');
+
+module.exports = ({ newsMapper }) => {
+  const create = data => Promise.resolve(data)
```
Comment (NikitaSedyx) [inline task_5/src/app/news/post.js]: didn't get why you wrap data in promise

File: task_5/src/infra/logger/logger.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+module.exports = () => ({
+  log: console.log,
```
Comment (NikitaSedyx) [inline task_5/src/infra/logger/logger.js]: why not to use winston, morgan etc?

File: task_5/src/infra/mappers/newsMapper.js [hunk 5]
```diff
@@ -0,0 +1,25 @@
+const { News: toEntity } = require('../../domain/news');
+
+module.exports = (storage) => {
```
Comment (NikitaSedyx) [inline task_5/src/infra/mappers/newsMapper.js]: will all your mappers receive storage as an argument?

File: task_5/src/app/news/post.js (hunk 3 above)
Comment (Killerkip10) [reply task_5/src/app/news/post.js]: done

File: task_5/src/infra/logger/logger.js (hunk 4 above)
Comment (Killerkip10) [reply task_5/src/infra/logger/logger.js]: done

File: task_5/src/app/news/get.js (hunk 2 above)
Comment (Killerkip10) [reply task_5/src/app/news/get.js]: done

File: task_5/config/index.js (hunk 1 above)
Comment (Killerkip10) [reply task_5/config/index.js]: done

**Pull request merged**